controller/backing_image_controller.go

```diff
@@ -431,6 +432,10 @@ func (bic *BackingImageController) handleBackingImageDataSource(bi *longhorn.Bac
 	}
 	existingBIDS := bids.DeepCopy()
 
+	if bids.Spec.UUID == "" {
+		bids.Spec.UUID = bi.Status.UUID
+	}
```

Comment:
Why not just handle this during the upgrade migration to make this change obvious because of new change introduced from 1.3.0?

Reply:
I don't know if users or others will unset this field accidentally hence I prefer to leave this check & update here.